faqs/galaxy/user_get_api_key.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Get API key on a Galayxy server
```

Comment:
I think this could probably use `preferences_admin_api_key.md`? Right now I know it has text for admin, but, there's already an if/else for dev, we could make it the same for admin maybe?

Reply:
I missed this snippet. I will use this one